osx/setup.sh

```diff
@@ -14,7 +14,7 @@
 # 9. Create the `<Xcode_Resources>/ka-lite/kalite/local_settings.py` based on `local_settings.default`.
 # 10. Copy the `ka-lite` and `pyrun` folders to the Xcode Resources folder.
 # 11. Build the Xcode project to produce the .app.
-# 12. Build the .dmg.
+# 12. Build the .mpkg.
```

Comment:
Please clean-up the comments because there are steps here that are no longer done like:

* # 6. Make ka-lite repo production-ready (delete .KALITE_SOURCE_DIR, etc).
* # 9. Create the `<Xcode_Resources>/ka-lite/kalite/local_settings.py` based on `local_settings.default`.

The download of the `assessment.zip` is also not included there.